Corona-Warn-App/src/main/java/de/rki/coronawarnapp/main/CWASettings.kt

```diff
@@ -24,11 +25,18 @@ class CWASettings @Inject constructor(
         get() = prefs.getBoolean(PKEY_DEVICE_TIME_INCORRECT_ACK, false)
         set(value) = prefs.edit { putBoolean(PKEY_DEVICE_TIME_INCORRECT_ACK, value) }
 
+    val lastAppVersion = prefs.createFlowPreference(
```

Comment:
As you noticed, we already store multiple onboarding true/false, and timestamps. All of which were not quite right to know when to show the new release infos. There may be multiple things we want to do on version upgrades.

Should we make this more specific? So there isn't any confusion what this reflects.
What do you think about:

```kotlin
val lastChangelogVersion = .....( key = "update.changelog.lastversion")
```

i.e. it's the last version for which the change log was shown.

Reply:
Yes, that's a fair point. I'll have it changed.